tutorials/ner_double/configs/double.cfg

```
@@ -0,0 +1,56 @@
+[paths]
+train = null
+dev = null
+
+[system]
+gpu_allocator = null
+
+[nlp]
+lang = "en"
+pipeline = ["ner_drug","ner"]
+batch_size = 1000
+
+[components]
+
+[components.ner]
+source = "en_core_web_md"
+
+[components.ner_drug]
+source = "en_ner_drugs"
+# Indicate the name of the model in the original pipeline
+component = "ner"
+# This model needs to use the tok2vec it was originally trained with
+replace_listeners = ["model.tok2vec"]
+
+[corpora]
+
+[corpora.train]
+@readers = "spacy.Corpus.v1"
+path = ${paths.train}
+max_length = 2000
+
+[corpora.dev]
+@readers = "spacy.Corpus.v1"
+path = ${paths.dev}
+max_length = 0
+
+[training]
+dev_corpus = "corpora.dev"
+train_corpus = "corpora.train"
+
+[training.optimizer]
+@optimizers = "Adam.v1"
+
+[training.batcher]
+@batchers = "spacy.batch_by_words.v1"
+discard_oversize = false
+tolerance = 0.2
+
+[training.batcher.size]
+@schedules = "compounding.v1"
+start = 100
+stop = 1000
+compound = 1.001
+
+[initialize]
+vectors = null
```

Ah, this will break silently after https://github.com/explosion/spaCy/pull/8388.
```suggestion
vectors = "en_core_web_md"
```

---

Well, not silently. You'll get a warning that you **shouldn't** ignore and you'll get bad predictions (silently).